### ISSUE TITLE
Ability to store and run snippets from local php files 

### ISSUE THREAD

**Issue by nickchomey** [sheabunge/code-snippets#131]
The main reason why I don't use Code Snippets is because it stores the snippets in the database. This prevents me from being able to use Xdebug for step debugging.

Instead I use the generally inferior, but sufficient, custom code tool within Toolset which stores its snippets in wp-content/toolset-customizations 

I'd love to have something similar with code snippets!

**Comment by sheabunge** [comment]
Thank you for the suggestion! It's something I've been wanting to implement for some time. 

Unfortunately it's not currently our first priority – some improvements to the block editor/elementor widgets are first to go, and then the first version of our cloud sync feature – but it's definitely high on my personal list of priorities. I'll post here again when I've had a chance to work out a prototype.

**Comment by ingenuitor** [comment]
> The main reason why I don't use Code Snippets is because it stores the snippets in the database. This prevents me from being able to use Xdebug for step debugging.
> 
> Instead I use the generally inferior, but sufficient, custom code tool within Toolset which stores its snippets in wp-content/toolset-customizations
> 
> I'd love to have something similar with code snippets!

@nickchomey @sheabunge I agree 100%, I'm about this idea, less database bloat, having everything static per say is always the best way for speed and code control, we already do this in our own custom application everything is static and it fly's like a no SQL db.
